bundles/binding/org.openhab.binding.dsmr/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="dsmr"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>DSMR Binding</name>
+    <description>The DSMR Binding obtains data from a Dutch smart meter ("Slimme meter" in Dutch) via the P1-port.</description>
+
```

Comment:
This touches one of the subjects I'd like to see addressed in do's and don'ts on the compatibility page.

Just making a binding compatible/installable with OH2 doesn't make me the author/maintainer. Also I do not know if the original author (@mvolaart) is still around to maintain the binding. I think it is bad etiquette to just put the name of someone else here without consulting them first. Maybe @mvolaart can tell us if he wants to be named as author here?

Furthermore, there are also bindings that are no longer supported by the original authors. Though people may make them compatible with OH2 and submit PRs for this. Who should then be the author? I've also read that being an author/maintainer has responsibilities such as making a binding roadmap and reviewing changes etc. The task should not be taken lightly. In the future there could bindings without an author/maintainer.

As a sidenode. I work in a settings where *@*author tags are considered as a bad practice because everybody owns the code ;-)